### PR TITLE
Migrate to new bootstrapping methods.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
This fixes https://github.com/flutter/devtools/issues/7444

* Moves bootstrapping logic into templated `flutter_bootstrap.js` file.
* Changes to `FlutterLoader.load()` API from `FlutterLoader.loadEntrypoint()` API.

### CODE REVIEW

File: packages/devtools_app/web/flutter_bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      canvasKitBaseUrl: 'canvaskit/'
+    }
+  });
+}
+
+// Bootstrap app for 1P environments:
+function bootstrapAppFor1P() {
+  _flutter.loader.load({
+    config: {
+      canvasKitBaseUrl: 'canvaskit/'
```
Comment (kenzieschmoll) [inline packages/devtools_app/web/flutter_bootstrap.js:40]: Looks like this is a change from the previous implementation, where we did not set the `canvasKitBaseUrl` field for 1P. @elliette do you remember why we needed separate bootstrapping for 1P? And does this look okay to you?

Comment (eyebrowsoffire) [reply packages/devtools_app/web/flutter_bootstrap.js:40]: Ah, that's just an oversight by me. I can remove that.

File: packages/devtools_app/web/flutter_bootstrap.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    config: {
+      canvasKitBaseUrl: 'canvaskit/'
+    }
+  });
+}
+
+// Bootstrap app for 1P environments:
+function bootstrapAppFor1P() {
+  _flutter.loader.load();
```
Comment (elliette) [inline packages/devtools_app/web/flutter_bootstrap.js:38]: I'm guessing we also need to specify the entrypointUrl as `main.dart.js`, see https://github.com/flutter/devtools/pull/5705/

File: packages/devtools_app/web/flutter_bootstrap.js (hunk 1 above)
Comment (elliette) [reply packages/devtools_app/web/flutter_bootstrap.js]: I forget why the `canvasKitBasUrl` is only required for 3P, but in general the assets are located in slightly different locations for 1P vs 3P which is why there are two different implementations.

File: packages/devtools_app/web/flutter_bootstrap.js (hunk 2 above)
Comment (eyebrowsoffire) [reply packages/devtools_app/web/flutter_bootstrap.js:38]: We should not need that, and the new API does not take an `entryPointUrl` as a parameter. The old API should have had that as the default and wouldn't have needed that anyway, so I'm not sure why that was needed before.

**Comment by elliette** [comment]
Note that we will need to update the DevTools copybara script as well, otherwise this will be break DevTools in google3. This is where we replace `bootstrapAppFor3P` with `bootstrapAppFor1P`: cl/513846717

**Review by elliette — APPROVED** [review]

**Pull request merged**